modules/tests/config/connection.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
hmm... i smell something here.. how about..
- rename this file with a `.template` extension/suffix
- add the original file name `modules/tests/config/connection.json` to the gitignore list
- and update readme to tell the developer to create a copy of the template file, remove the extension and add in values relating to his/her/their environment

Reply:
You seem to have a special nose! @ChintanRaval

Reply:
👃